src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,375 @@
+        ServiceService.getDeployedServiceDetailsById(uuid, localStorage.getItem(usernameKey)!)
+            .then((response) => {
+                setRequestSubmitted(false);
+                if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_SUCCESS) {
+                    setTip(
+                        OrderSubmitResult(
+                            ProcessingStatus(response, OperationType.destroy as OperationType),
+                            uuid,
+                            'success'
+                        )
+                    );
+                    setDestroying(false);
+                    TipClear();
+                    getCloseStatus(true);
+                } else if (response.serviceState === ServiceDetailVo.serviceState.DESTROY_FAILED) {
+                    setTip(
+                        OrderSubmitResult(
+                            ProcessingStatus(response, OperationType.destroy as OperationType),
+                            uuid,
+                            'error'
+                        )
+                    );
+                    setDestroying(false);
+                    TipClear();
+                } else {
+                    setTimeout(() => {
+                        waitingServiceDestroy(uuid, timeout - waitServicePeriod, date);
+                    }, waitServicePeriod);
+                }
+            })
+            .catch((e: Error) => {
+                setTip(OrderSubmitResult('Destroy failed:' + e.message, uuid, 'error'));
+                setDestroying(false);
+                TipClear();
+                setRequestSubmitted(false);
+            });
+    }
+
+    const steps = [
+        {
+            title: 'Export data',
+            content: <ExportServiceData />,
+            description: 'Export service data.',
+        },
+        {
+            title: 'Select a destination',
+            content: (
+                <SelectDestination
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    getSelectedParameters={getSelectedParameters}
+                    currentCsp={selectCsp}
+                    currentArea={selectArea}
+                    currentRegion={selectRegion}
+                    currentFlavor={selectFlavor}
+                />
+            ),
+            description: 'Select a destination for deployment.',
+        },
+        {
+            title: 'Deploy',
+            content: (
+                <ShowDeploy
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    selectCsp={selectCsp}
+                    selectArea={selectArea}
+                    selectRegion={selectRegion}
+                    selectFlavor={selectFlavor}
+                    onFinish={handleFinish}
+                    deploying={deploying}
+                    formRef={formRef}
+                />
+            ),
+            description: 'Deploy the service.',
```

Comment:
Deploy the service on the new destination.

Reply:
fixed.